src/ui/components/SecondaryToolbox/ReactDevTools.tsx

```diff
@@ -1,4 +1,5 @@
 import { NodeBounds, Object as ProtocolObject } from "@replayio/protocol";
+import { createBridge, createStore, initialize } from "@replayio/react-devtools-inline/frontend";
```

Comment:
:hand: Hmm. I'm going to half-rethink my previous comment.

I still think we don't need to worry about the RDT backend version at this point, it's always going to be the "modern" version.

But, this does now switch from a lazy import of `react-devtools-inline` (and thus hopefully a separate chunk only loaded on demand) to _always_ loading it.

With Next's build setup, I don't know if that `await import("@replayio/react-devtools-inline/frontend")` was actually getting written to a separate chunk, or if it was getting inlined into the larger frontend bundle.  I'd have to run a bundle analysis to check.

We haven't spent any time lately on trying to optimize JS bundle sizes or page loads.  Honestly this probably is fine for now and we can do bundle size analysis later - I'm just noting that this _is_ a shift in how we import the package which could result in the main app bundle being larger.

Reply:
Yeah, FWIW we lazily import the root `ReactDevTools` component:
```js
const ReactDevToolsPanel = React.lazy(() => import("./ReactDevTools"));
```

This should decouple all of that logic from the main bundle, so far as I understand. Looking at my fork, I can verify that two JS files (~236kb and ~53kb) are downloaded only once the React tab has been clicked and opened.